apps/backoffice-v2/src/domains/business-reports/adapters/report-adapter/report-adapter.ts

```diff
@@ -97,22 +97,20 @@ export const reportAdapter = {
       relatedAdsImages: Object.values(data?.report?.data?.socialMedia?.ads ?? {}).map(
         data => data?.pickedAd?.imageUrl,
       ),
-      onlineReputationAnalysis: data?.report?.data?.websiteScamAnalysis?.riskIndicators?.map(
```

Comment:
Callback data uses the names of the PDFs as the property keys, here its transactionLaundering.